### ISSUE TITLE
System tray/indicator does not display on Ubuntu

### ISSUE THREAD

**Issue by netlore** [gtkevemon/gtkevemon#55]
My understanding is that there is a legacy API for this which has now been deprecated in Ubuntu in favour of the freedesktop standard for application indicators.... 

It would be very nice if this worked correctly on Ubuntu.

**Comment by simonfuhrmann** [comment]
GtkEveMon uses the Gtkmm StatusIcon which follows the freedesktop.org "System Tray" specification.
https://developer.gnome.org/gtkmm/stable/classGtk_1_1StatusIcon.html

I cleaned up the code a bit, behavior might have changed. See if it is working for you now. Also make sure you enable tray usage in the configuration. Since my window manager (E17) behaves strangely when using the skip_taskbar_hint (so that the taskbar item disappears when in tray), I disabled that for now. Feel free to play with the code and present a better solution :)
